src/components/coin-row/CoinRowInfoButton.js

```diff
@@ -93,7 +93,7 @@ const CoinRowInfoButton = ({ item, onCopySwapDetailsText, showFavoriteButton })
   );
 
   const onPressAndroid = useCallback(() => {
-    const blockExplorerText = `View on ${startCase(ethereumUtils.getBlockExplorer(item?.chainId))}`;
+    const blockExplorerText = `View on ${startCase(ethereumUtils.getBlockExplorer({ chainId: item?.chainId }))}`;
```

Comment:
on line 72 of this file, `blockExplorerName: startCase(ethereumUtils.getBlockExplorer(chainId)),` is passing `chainId` directly instead of `{ chainId }`